features/storage/csi_clone.feature

```diff
@@ -183,3 +183,89 @@ Feature: CSI clone testing related feature
     Then the step should succeed
     And the output should contain "test data"
 
+  # @author wduan@redhat.com
+  # @case_id OCP-27617
+  @admin
+  @destructive
+  Scenario: [Cinder CSI Clone] Clone a pvc with default storageclass
+    Given default storage class is patched to non-default
+    And admin clones storage class "my-csi-default" from "standard-csi" with:
+      | ["metadata"]["name"] | my-csi-default |
+    When I run the :patch admin command with:
+      | resource      | storageclass                                                                           |
+      | resource_name | my-csi-default                                                                         |
+      | p             | {"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}} |
+
+    # Create mypvc-ori without sc specified
+    Given I have a project
+    When I obtain test data file "storage/misc/pvc.json"
+    When I create a dynamic pvc from "pvc.json" replacing paths:
+      | ["metadata"]["name"] | mypvc-ori |
+    Then the step should succeed
+    Given I obtain test data file "storage/misc/pod.yaml"
+    When I run oc create over "pod.yaml" replacing paths:
+      | ["metadata"]["name"]                                         | mypod-ori  |
+      | ["spec"]["volumes"][0]["persistentVolumeClaim"]["claimName"] | mypvc-ori  |
+      | ["spec"]["containers"][0]["volumeMounts"][0]["mountPath"]    | /mnt/local |
+    Then the step should succeed
+    Given the pod named "mypod-ori" becomes ready
+    When I execute on the pod:
+      | sh | -c | echo "clone test" > /mnt/local/testfile |
+    Then the step should succeed
+    And I execute on the pod:
+      | sh | -c | sync -f /mnt/local/testfile |
+    Then the step should succeed
+
+    # Clone mypvc-ori without sc specified
+    Given I obtain test data file "storage/csi/pvc-clone.yaml"
+    When I create a dynamic pvc from "pvc-clone.yaml" replacing paths:
+      | ["metadata"]["name"]                         | mypvc-clone  |
```

Comment:
Please fix the extra spaces issue in the tables.
LGTM other than that.

Reply:
Fixed.